### DISCUSSION TITLE
Running Prisma In Digital Ocean Droplet

### DISCUSSION THREAD

**Discussion by Anan7Codes** [prisma/prisma#8445] [Q&A] [answered]
Hey, I'm getting an Error: P1001 on "npx prisma instrospect" in my vps. Can't reach database server apparently. I'm using hostinger's remote mysql right now and locally it works fine. I'm using the same .env file. It worked fine before idk why its not working now thoh.

**Comment by ryands17** [comment, marked as answer]
@Anan7Codes 👋 

Is the database accessible from your VPS? Was there any change in permissions or something like that?

**Reply by Anan7Codes** [reply]
It worked initially. Now its not *facepalm*. My bad.

**Reply by ryands17** [reply]
So is it working now?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
